src/main/java/me/botsko/prism/commands/FlagsCommand.java

```diff
@@ -36,7 +36,7 @@ private void help(CommandSender sender) {
 		sender.sendMessage(Prism.messenger.playerMsg(
 				ChatColor.GRAY + "Flags control how Prism applies a rollback/restore, or formats lookup results."));
 		sender.sendMessage(Prism.messenger
-				.playerMsg(ChatColor.GRAY + "Use them after parameters, like /pr l p:<player_name> -extended"));
+				.playerMsg(ChatColor.GRAY + "Use them after parameters, like /pr l p:[player] -extended"));

```

Comment:
Why the change? Standard syntax would be `<player>` for required values. Typically `[player]` indicates optional values.

Reply:
I searched all of the files, and there is a mix of `[player]`
* https://github.com/AddstarMC/Prism-Bukkit/blob/master/src/main/java/me/botsko/prism/commands/ParamsCommand.java#L41
* https://github.com/AddstarMC/Prism-Bukkit/blob/master/src/main/java/me/botsko/prism/commands/ParamsCommand.java#L60

and  `<player>`
* https://github.com/AddstarMC/Prism-Bukkit/blob/master/src/main/java/me/botsko/prism/commands/ParamsCommand.java#L40
* https://github.com/AddstarMC/Prism-Bukkit/blob/master/src/main/java/me/botsko/prism/commands/FlagsCommand.java#L39

I'd be happy though, to switch them all to `<player>` -- I don't care either way.  I just noticed that the previous commit switched this one to `<player>`

Reply:
Oh! I just realized your delineation between standard and optional.  Perhaps that's why there is a mix.  So, I'll back out that change to `[player]` and instead change
` Base the radius around another place, like r:[player]:20 or r:20,35,10:5 (x,y,z)`
to
` Base the radius around another place, like r:player:20 or r:20,35,10:5 (x,y,z)`

Reply:
Yeah Viv used [player] - I had used <player_name> as it was more standard in mc help

Reply:
I'm not too fussed either, just thought it was worth pointing out the standard command line convention (in general, not with Prism) is to show required fields with `<player>` and optional with `[player]` :)